src/Bicep.Cli/Program.cs

```diff
@@ -37,6 +39,10 @@ public Program(IResourceTypeProvider resourceTypeProvider, TextWriter outputWrit
 
         public static int Main(string[] args)
         {
+            string profilePath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), LanguageConstants.LanguageId);
```

Comment:
>LocalApplicationData [](http://example.com/codeflow?start=98&length=20)

Not sure about linux or mac, but on windows you can launch a process without loading the user profile, which then won't have local app data. It might be worth choosing a different path or wrapping this in an exception handler so we can function normally in that case.

Reply:
I believe either Batch or Azure Functions launch processes this way.

Reply:
Solved this via the other comment.